index.html

```diff
@@ -0,0 +1,340 @@
+            <h2 class="features__heading container">Features</h2>
+            <div class="features__content container">
+                <div class="feature">
+                    <div class="icon__container">
+                        <img
+                            class="feature__icon"
+                            src="./images/responsive.svg"
+                            alt="mobile-phone"
+                        >
+                    </div>
+                    <h4>Fully responsive</h4>
+                    <p>Looks amazing on any device, smartphone, tablet, laptop and desktop.</p>
+                </div>
+                <div class="feature">
+                    <div class="icon__container">
+                        <img
+                            class="feature__icon"
+                            src="./images/customizable.svg"
+                            alt="tools"
+                        >
+                    </div>
+                    <h4>Customizable</h4>
+                    <p>Change the colors, pictures or any of the sections to suit your needs.</p>
+                </div>
+                <div class="feature">
+                    <div class="icon__container">
+                        <img
+                            class="feature__icon"
+                            src="./images/design.svg"
+                            alt="certificate"
+                        >
+                    </div>
+                    <h4>Slick and beautiful design</h4>
+                    <p>Trendy and fresh design, fits any website.</p>
+                </div>
+            </div>
+
+        </section>
+
+        <section class="logos main-container">
+            <div class="logos__images container">
+                <div class="logos__images-container">
+
+                </div>
+            </div>
+
+        </section>
+
+        <section class="about main-container">
+            <div class="about__image">
+                <div class="about__image-container">
+
+                </div>
+            </div>
+            <div class="about__info">
+                <h1 class="about__header">Simple and beautiful</h1>
+                <p class="about__sub-header">
+                    Use Nova theme for your next project. It is completely customizable so you
+                    can change any of the section to fit your needs. Nova Theme is Free for any kind of use, personal
+                    and commercial. Have fun and good luck!
+                </p>
+            </div>
+
+        </section>
+
+        <section
+            id="offer"
+            class="offer main-container"
+        >
+            <div class="offer__heading container">
+                <h2 class="offer__header">Choose your pricing plan</h2>
+                <p class="offer__sub-header">Pick any our super affordable pricing plans</p>
+            </div>
+            <div class="offer__plans container">
+                <div class="offer__card offer__card--shadow">
```

Comment:
Mamy tutaj "listę" pakietów cenowych więc ponownie użyłbym `ul > li`, gdzie li to każdy z pakietu :)